spec/models/intake_spec.rb

```diff
@@ -841,46 +841,47 @@
         end
 
         it "assigns to the correct group and the correct instance" do
-          expect(intake.reload.vita_partner_group_id).to eq EitcZendeskInstance::ONLINE_INTAKE_UW_CENTRAL_OHIO
+          expect(intake.reload.vita_partner.name).to eq "United Way of Central Ohio"
           expect(intake.zendesk_instance).to eq EitcZendeskInstance
         end
       end
     end
 
     context "with state routing" do
-      shared_examples "state-level routing" do |state_criteria, zendesk_group_id, zendesk_instance|
+      shared_examples "state-level routing" do |state_criteria, partner_name|
         context "given a state" do
           let(:state) { state_criteria } # might not be necessary?
+          let(:partner) { VitaPartner.find_by!(name: partner_name) }
```

Comment:
Adding an exception here--especially given that it's a shared example set--is an excellent choice.